src/features/flows/commonResources.ts

```diff
@@ -80,7 +82,7 @@ export const matterDevResources: ResourcesWithdDownloadAndGuide[] = [
         },
     },
     {
-        label: 'CHIP Cert application',
+        label: 'CHIP certification application',
```

Comment:
Hmm, this application actually is named "Chip Cert" and it is dedicated to certification. Should we use the original app name, or, as you suggested "what this application is doing"?

Reply:
Note that I keep the name later on in lines 89 and 93. Here, I just thought we can be a bit more semantically open, so to speak :D